README.md

```diff
@@ -6,8 +6,9 @@ DISCLAIMER: This addon is not actively maintained for public use. Pull requests
 Compatibility
 ------------------------------------------------------------------------------
 
-* Ember.js v2.18 or above
+* Ember.js v3.8 or above
 * Ember CLI v2.13 or above
+* Node.js v8 or above
 
 
 Installation
```

Comment:
we should update the README documentation to:
- remove section about "classic class" mixin usage
- remove section about "...use the route-pagination as a decorator"
- remove other references to "decorators", and update code snippet import paths to be `@gavant/ember-pagination/mixins` instead of `@gavant/ember-pagination/decorators`

Reply:
Does this mean I should delete the decorator files from the repo?